### PR TITLE
Updated xml ws spec for 3.0 release

### PR DESCRIPTION
**Pull request by cesarhernandezgt**

### CODE REVIEW

File: spec/pom.xml [hunk 1]
```diff
@@ -28,18 +28,16 @@
...
         <maven.site.skip>true</maven.site.skip>
-        <asciidoctor.maven.plugin.version>1.5.7.1</asciidoctor.maven.plugin.version>
-        <asciidoctorj.version>1.6.2</asciidoctorj.version>
-        <asciidoctorj.pdf.version>1.5.0-alpha.16</asciidoctorj.pdf.version>
-        <jruby.version>9.2.6.0</jruby.version>
+        <asciidoctor.maven.plugin.version>2.0.0</asciidoctor.maven.plugin.version>
+        <asciidoctorj.pdf.version>1.5.3</asciidoctorj.pdf.version>
         <!-- status: DRAFT, BETA, etc., or blank for final -->
-        <status>DRAFT</status>
+        <status>Final Release</status>
```
Comment (lukasj) [inline spec/pom.xml:40]: This should be unchanged. To build final version, just pass `-Dstatus=""` to the build of the spec project, see the comment above this line.

File: spec/src/main/asciidoc/license-efsl.adoc [hunk 2]
```diff
@@ -14,7 +14,7 @@ endif::[]
 Release: {revdate}
 ....
 
-Copyright (c) 2019 Eclipse Foundation.
+Copyright (c) 2020 Eclipse Foundation.
```
Comment (lukasj) [inline spec/src/main/asciidoc/license-efsl.adoc:17]: the work has been started in 2019, so the header should say `2019, 2020`

**Comment by lukasj** [comment]
note that some changes you've proposed came in through #135

**Comment by cesarhernandezgt** [comment]
thanks for the feedback @lukasj , I'm closing this PR in favor of https://github.com/eclipse-ee4j/jax-ws-api/pull/135

**Pull request closed without merging**